### PR TITLE
Create the system's front-end

### PR DESCRIPTION
**Pull request by RonnyFrayRegato**
Create the system's front-end, including React components, routing, and API calls.

### CODE REVIEW

File: phit-ui/src/pages/home-page/home-page.css
Comment (AlexisMLopez) [inline phit-ui/src/pages/home-page/home-page.css:1]: Remove unnecessary newlines on lines 1, 24, 33 and 65 to improve readability.

File: phit-ui/src/pages/home-page/home-page.js
Comment (AlexisMLopez) [inline phit-ui/src/pages/home-page/home-page.js:1]: Add a 'label' for each input element to improve accessibility.
Change the type of the password input to "password" for better security.
Change the case of Username and Password to lowercase, following JavaScript naming conventions.
Adjust formatting to comply with HTML and JavaScript general style guidelines.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
+
+a:visited {
+    color: black;
+}
+
+a:link {
+    text-decoration: none;
+    color: black;
+}
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css]: Add newline to comply with general CSS style guidelines.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css]: Remove newline to comply with general CSS style guidelines.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    padding-top: 2vh;
+    font-size: 4vh;
+}
+
+.medical-admin-allergies-header-text {
+    padding-top: 2vh;
+    margin-left: 2vh;
+}
+
+.medical-admin-allergies-body {
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css]: Remove dead code on lines 51-53 to improve readability.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    border: 3px solid black;
+    background-color: darkgray;
+    border-radius: 50%;
+}
+
+.medical-admin-allergies-exit-text {
+    padding-top: 22.2vh;
+    font-size: 4vh;
+}
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css]: Remove newline to comply with general CSS style guidelines.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+.table-loading{
+    color:black;
+    padding-left: 83vh;
+}
+.login-input {
+    padding-top: 20px;
+    text-align:center;
+    font-size: 20px;
+    display: block;
+}
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.css]: Add newline at end of file.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.js
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-allergies.js:1]: Remove newlines on lines 8, 35, 46, 51, 54, 56, 61, 63, 88, and 99.
Update formatting to comply with HTML and JavaScript general style guidelines.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css [hunk 6]
```diff
@@ -0,0 +1,150 @@
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css]: Remove newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+a:visited {
+    color: black;
+}
+
+a:link {
+    text-decoration: none;
+    color: black;
+}
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css]: Add newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+    font-size: 4vh;
+}
+
+.medical-admin-medications-header-text {
+    padding-top: 2vh;
+    margin-left: 2vh;
+}
+
+.medical-admin-medications-body {
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css]: Remove newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+    border: 3px solid black;
+    background-color: darkgray;
+    border-radius: 50%;
+}
+
+.medical-admin-medications-exit-text {
+    padding-top: 22.2vh;
+    font-size: 4vh;
+}
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css]: Remove newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+    padding-top: 20px;
+    text-align:center;
+    font-size: 20px;
+    display: block;
+}
+
+.table-loading{
+    color:black;
+    padding-left: 83vh;
+}
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css]: Add newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+.login-input {
+    padding-top: 20px;
+    text-align:center;
+    font-size: 20px;
+    display: block;
+}
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.css]: Add newline at end of file.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css [hunk 12]
```diff
@@ -0,0 +1,144 @@
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css]: Remove newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css [hunk 13]
```diff
@@ -0,0 +1,144 @@
...
+a:visited {
+    color: black;
+}
+
+a:link {
+    text-decoration: none;
+    color: black;
+}
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css]: Add newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css [hunk 14]
```diff
@@ -0,0 +1,144 @@
...
+    font-size: 4vh;
+}
+
+.medical-admin-patients-header-text {
+    padding-top: 2vh;
+    margin-left: 2vh;
+}
+
+.medical-admin-patients-body {
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css]: Remove newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css [hunk 15]
```diff
@@ -0,0 +1,144 @@
...
+    border: 3px solid black;
+    background-color: darkgray;
+    border-radius: 50%;
+}
+
+.medical-admin-patients-exit-text {
+    padding-top: 22.2vh;
+    font-size: 4vh;
+}
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css]: Remove newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css [hunk 16]
```diff
@@ -0,0 +1,144 @@
...
+    padding-top: 20px;
+    text-align:center;
+    font-size: 20px;
+    display: block;
+}
+
+.table-loading{
+    color:black;
+    padding-left: 83vh;
+}
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.css]: Add newline at end of file.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css [hunk 17]
```diff
@@ -0,0 +1,145 @@
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css]: Remove newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css [hunk 18]
```diff
@@ -0,0 +1,145 @@
...
+a:visited {
+    color: black;
+}
+
+a:link {
+    text-decoration: none;
+    color: black;
+}
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css]: Add newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css [hunk 19]
```diff
@@ -0,0 +1,145 @@
...
+    font-size: 4vh;
+}
+
+.medical-admin-supply-header-text {
+    padding-top: 2vh;
+    margin-left: 2vh;
+}
+
+.medical-admin-supply-body {
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css]: Remove newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css [hunk 20]
```diff
@@ -0,0 +1,145 @@
...
+    padding-bottom: 5vh;
+    background-color: #D9D9D9;
+    color: black;
+}
+
+.medical-admin-supply-table-bottom-spacer {
+    margin-left: auto;
+    margin-right: auto;
+    padding-left: 30vh;
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css]: Remove newline.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css [hunk 21]
```diff
@@ -0,0 +1,145 @@
...
+    border: 3px solid black;
+    background-color: darkgray;
+    border-radius: 50%;
+}
+
+.medical-admin-supply-exit-text {
+    padding-top: 22.2vh;
+    font-size: 4vh;
+}
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.css]: Remove newline.

File: phit-ui/src/pages/system-admin-page/system-admin-home-page.css [hunk 22]
```diff
@@ -0,0 +1,156 @@
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/system-admin-page/system-admin-home-page.css]: Remove newline.

File: phit-ui/src/pages/system-admin-page/system-admin-home-page.css [hunk 23]
```diff
@@ -0,0 +1,156 @@
...
+a:visited {
+    color: black;
+}
+
+a:link {
+    text-decoration: none;
+    color: black;
+}
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/system-admin-page/system-admin-home-page.css:8]: Add newline.

File: phit-ui/src/pages/system-admin-page/system-admin-home-page.css [hunk 24]
```diff
@@ -0,0 +1,156 @@
...
+a:hover {
+    color: #6FA8FF;
+}
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/system-admin-page/system-admin-home-page.css:13]: Remove newline.

File: phit-ui/src/pages/system-admin-page/system-admin-home-page.css [hunk 25]
```diff
@@ -0,0 +1,156 @@
...
+    padding-top: 5vh;
+    padding-left:10vh;
+    color:black;
+}
+
+.page-title {
+    text-align: center;
+    font-size: 6vh;
+}
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/system-admin-page/system-admin-home-page.css:61]: Remove newline.

File: phit-ui/src/pages/system-admin-page/system-admin-home-page.css [hunk 26]
```diff
@@ -0,0 +1,156 @@
...
+    height: 1vh;
+    background-color: #D9D9D9;
+    color: black;
+}
+
+.system-admin-home-table-wrapper {
+    overflow-y: auto;
+    height: 60vh;
+}
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/system-admin-page/system-admin-home-page.css:93]: Remove newline.

File: phit-ui/src/pages/system-admin-page/system-admin-home-page.css [hunk 27]
```diff
@@ -0,0 +1,156 @@
...
+tr:nth-child(even) {
+    background-color: #D9D9D9;
+}
+
+td {
+    font-size: 2vh;
+    text-align: center;
+}
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/system-admin-page/system-admin-home-page.css:107]: Remove newline.

File: phit-ui/src/pages/system-admin-page/system-admin-home-page.css [hunk 28]
```diff
@@ -0,0 +1,156 @@
...
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/system-admin-page/system-admin-home-page.css]: Remove newline.

File: phit-ui/src/pages/system-admin-page/system-admin-home-page.css [hunk 29]
```diff
@@ -0,0 +1,156 @@
...
+.column {
+    padding: 0 1vh;
+    display:flex;
+}
+
+.column img {
+    margin-top: 5vh;
+    vertical-align: middle;
+    margin-left: 5vw;
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/system-admin-page/system-admin-home-page.css]: Remove newline.

File: phit-ui/src/pages/system-admin-page/system-admin-home-page.css [hunk 30]
```diff
@@ -0,0 +1,156 @@
...
+
```
Comment (AlexisMLopez) [inline phit-ui/src/pages/system-admin-page/system-admin-home-page.css]: Remove newline.

**Review by AlexisMLopez — CHANGES_REQUESTED** [review]
The code needs improvement in several areas to make it more readable and easier to understand. Inconsistent use of white-space and indentation makes it hard to follow the code flow. Additionally, variable names have inconsistent naming conventions, which can be confusing for other developers to understand their purpose. Adding comments and documentation is necessary to clarify the code's functionality, making it easier for others to maintain it in the future.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.js
Comment (zachary-mixson) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-medications.js:1]: Remove newlines on lines 8, 35, 46, 51, 54, 56, 61, 63, 88, and 99.
Update formatting to comply with HTML and JavaScript general style guidelines.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.js
Comment (zachary-mixson) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-patients.js:1]: Remove newlines on lines 8, 35, 46, 51, 54, 56, 61, 63, 88, and 99.
Update formatting to comply with HTML and JavaScript general style guidelines.

File: phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.js
Comment (zachary-mixson) [inline phit-ui/src/pages/medical-admin-page/medical-admin-page-supply.js:1]: Remove newlines on lines 8, 35, 46, 51, 54, 56, 61, 63, 88, and 99.
Update formatting to comply with HTML and JavaScript general style guidelines.

**Review by zachary-mixson — CHANGES_REQUESTED** [review]
fix styling issues.

**Review by zachary-mixson — APPROVED** [review]

File: phit-ui/.gitignore [hunk 31]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (RonnyFrayRegato) [inline phit-ui/.gitignore:1]: @htbryant1256 Add .vs to the .gitignore.

**Review by AlexisMLopez — APPROVED** [review]
